### DISCUSSION TITLE
Do not send SR(Sender Report) RTCP feedback message to server

### DISCUSSION THREAD

**Discussion by flying1314** [ossrs/ffmpeg-webrtc#9] [Ideas]
Currently not send SR to the RTC server which used it to synchronize audio and video to the same timebase

**Comment by winlinvip** [comment]
We are currently working on implementing basic protocols and frameworks. In the future, RTCP SR will be supported. Additionally, OBS WHIP has not been implemented yet, which means we cannot convert from RTC to RTMP at the moment.

**Reply by flying1314** [reply]
OBS has support RTCP SR since libdatachannel support it.

**Reply by winlinvip** [reply]
Yes, it has been confirmed that OBS 30.0.2 supports sending RTCP SR packets.
